README.md

```diff
@@ -50,10 +50,8 @@ The following workflow is for a new **Helm** operator:
 First, checkout and install the operator-sdk CLI:
 
 ```sh
-$ mkdir -p $GOPATH/src/github.com/operator-framework
-$ cd $GOPATH/src/github.com/operator-framework
-$ git clone https://github.com/operator-framework/operator-sdk
-$ cd operator-sdk
+$ go get github.com/operator-framework/operator-sdk
```

Comment:
While `go get` technically works, it does seem to have the side effect that it tries to build and install the repo. Since we're using a `Makefile` the resulting `go get` build may fail.

```
$ go get github.com/operator-framework/operator-sdk
package github.com/operator-framework/operator-sdk: no Go files in /home/devel/jesusr/dev/src/github.com/operator-framework/operator-sdk```

The `mkdir` and `git clone` are more explicit as we do not conform to what `go get` would expect when downloading and building a go project. I find that misleading.

Reply:
to avoid installation of code, we can add -d in the "go get " command